packages/table/src/common/_variables.scss

```diff
@@ -49,3 +49,9 @@ $resize-horizontal-cursor: ns-resize !default;
 
 // Actions
 $action-cursor: pointer !default;
+
+/*
+Z-index layers
+*/
+$interactive-cell-z-index: 10;
+$region-layer-z-index: 9;
```

Comment:
> You can assign to variables if they aren’t already assigned by adding the !default flag to the end of the value. This means that if the variable has already been assigned to, it won’t be re-assigned, but if it doesn’t have a value yet, it will be given one.